src/window.rs

```diff
@@ -0,0 +1,147 @@
+use std::sync::{Arc, Mutex, RwLock};
+use std::sync::mpsc;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::io;
+use std::thread::{spawn, JoinHandle};
+use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr, UdpSocket};
+use result::{Error, Result};
+
+const NUM_BLOCKS: usize = 10 * 2048;
+const BLOCK_SIZE: usize = 64 * 1024;
+
+#[derive(Clone)]
+pub struct Block {
```

Comment:
`Block` is used in the `Ledger` module. Let's be very particular about the use of this term. Either `ledger::Block` should change or this one. Which do you think is closest to the concept of a Block in our streaming protocol?

Reply:
doh, this file should be removed!